### ISSUE TITLE
configuring flow.generator to mermaid seems broken, no pictures are generated

### ISSUE THREAD

**Issue by checkiecheck** [frankframework/frankframework#5739]
For SittardGeleen we wanted to show an other image generator for assessment. However setting the flow.generator property to MermaidFlowGenerator (or full classpath), only throws an exception at startup and no images show up in the console.

```

**Input**
```
2023-11-13 11:50:03,137 WARN [main] {} flow.FlowDiagramManager - FlowGenerator class [MermaidFlowGenerator] not found
java.lang.ClassNotFoundException: MermaidFlowGenerator
      at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1412) ~[catalina.jar:9.0.74]
      at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1220) ~[catalina.jar:9.0.74]
      at nl.nn.adapterframework.util.ClassUtils.loadClass(ClassUtils.java:143) ~[ibis-adapterframework-commons-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.util.flow.FlowDiagramManager.createFlowGenerator(FlowDiagramManager.java:90) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.util.flow.FlowDiagramManager.afterPropertiesSet(FlowDiagramManager.java:73) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireByName(AbstractAutowireCapableBeanFactory.java:1471) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1413) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:378) ~[spring-beans-5.3.30.jar:5.3.30]
      at nl.nn.adapterframework.util.SpringUtils.createBean(SpringUtils.java:26) ~[ibis-adapterframework-commons-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.management.bus.MessageDispatcher.findServiceActivators(MessageDispatcher.java:99) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.management.bus.MessageDispatcher.afterPropertiesSet(MessageDispatcher.java:69) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:921) [spring-context-5.3.30.jar:5.3.30]
      at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583) [spring-context-5.3.30.jar:5.3.30]
      at nl.nn.adapterframework.lifecycle.IbisApplicationContext.createApplicationContext(IbisApplicationContext.java:109) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:116) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:96) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.lifecycle.IbisApplicationServlet.init(IbisApplicationServlet.java:80) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at javax.servlet.GenericServlet.init(GenericServlet.java:158) [servlet-api.jar:4.0.FR]
      at org.apache.catalina.core.StandardWrapper.initServlet(StandardWrapper.java:1106) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:1063) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:960) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4673) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4980) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:683) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:658) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:662) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1184) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1932) [catalina.jar:9.0.74]
      at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_392]
      at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_392]
      at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.74]
      at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:112) [?:1.8.0_392]
      at org.apache.catalina.startup.HostConfig.deployDirectories(HostConfig.java:1094) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:476) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1617) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:318) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:898) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:795) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1332) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1322) [catalina.jar:9.0.74]
      at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_392]
      at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.74]
      at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:134) [?:1.8.0_392]
      at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:871) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:249) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardService.startInternal(StandardService.java:428) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:917) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.Catalina.start(Catalina.java:772) [catalina.jar:9.0.74]
      ... suppressed 4 lines
      at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:347) [bootstrap.jar:9.0.74]
      at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:478) [bootstrap.jar:9.0.74]
```

**Environment**
```
FF! 7.9-20231028.143509: Frank2SittardGeleen
Running on LAPTOP-TDT0D8E using Apache Tomcat/9.0.74
Java Version: OpenJDK Runtime Environment (1.8.0_392-b08)
Heap size: 239M, total JVM memory: 696M
Free disk space: 302GB, total disk space: 466GB
Up since: 2023-11-13 12:22:46 (14m)```

**Additional Environment**
NA

**Additional Context**

**Comment by nielsm5** [comment]
Staat er niet meer in de log? Deze stacktrace hebben we niet zo veel aan...

**Comment by checkiecheck** [comment]
`2023-11-13 11:49:56,062 DEBUG [main] {} APPLICATION - Starting IBIS WebApplicationInitializer
2023-11-13 11:49:56,129 INFO [main] {} APPLICATION - Determined ApplicationServer [TOMCAT]
2023-11-13 11:50:02,285 INFO [main] {} APPLICATION - Successfully started IBIS WebApplicationInitializer
2023-11-13 11:50:02,312 INFO [main] {} APPLICATION - IbisApplicationServlet starting IbisContext
2023-11-13 11:50:02,316 DEBUG [main] {} APPLICATION - Starting application [Frank2SittardGeleen]
2023-11-13 11:50:02,317 DEBUG [main] {} APPLICATION - Creating IbisApplicationContext
2023-11-13 11:50:02,318 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-akamai] version [7.9-20231028.143509]
2023-11-13 11:50:02,320 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-aspose] version [7.9-20231028.143509]
2023-11-13 11:50:02,321 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-aws] version [7.9-20231028.143509]
2023-11-13 11:50:02,322 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-cmis] version [7.9-20231028.143509]
2023-11-13 11:50:02,324 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-commons] version [7.9-20231028.143509]
2023-11-13 11:50:02,325 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-console-backend] version [7.9-20231028.143509]
2023-11-13 11:50:02,326 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-coolgen] version [7.9-20231028.143509]
2023-11-13 11:50:02,327 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-core] version [7.9-20231028.143509]
2023-11-13 11:50:02,328 DEBUG [main] {} APPLICATION - Loading IAF module [credentialprovider] version [7.9-20231028.143509]
2023-11-13 11:50:02,329 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-ibm] version [7.9-20231028.143509]
2023-11-13 11:50:02,331 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-ifsa] version [7.9-20231028.143509]
2023-11-13 11:50:02,332 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-ladybug] version [7.9-20231028.143509]
2023-11-13 11:50:02,333 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-larva] version [7.9-20231028.143509]
2023-11-13 11:50:02,334 DEBUG [main] {} APPLICATION - Loading IAF module [iaf-management-gateway] version [7.9-20231028.143509]
2023-11-13 11:50:02,336 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-sap] version [7.9-20231028.143509]
2023-11-13 11:50:02,337 DEBUG [main] {} APPLICATION - Loading IAF module [ibis-adapterframework-tibco] version [7.9-20231028.143509]
2023-11-13 11:50:03,137 WARN [main] {} flow.FlowDiagramManager - FlowGenerator class [MermaidFlowGenerator] not found
java.lang.ClassNotFoundException: MermaidFlowGenerator
      at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1412) ~[catalina.jar:9.0.74]
      at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1220) ~[catalina.jar:9.0.74]
      at nl.nn.adapterframework.util.ClassUtils.loadClass(ClassUtils.java:143) ~[ibis-adapterframework-commons-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.util.flow.FlowDiagramManager.createFlowGenerator(FlowDiagramManager.java:90) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.util.flow.FlowDiagramManager.afterPropertiesSet(FlowDiagramManager.java:73) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireByName(AbstractAutowireCapableBeanFactory.java:1471) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1413) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:619) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:378) ~[spring-beans-5.3.30.jar:5.3.30]
      at nl.nn.adapterframework.util.SpringUtils.createBean(SpringUtils.java:26) ~[ibis-adapterframework-commons-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.management.bus.MessageDispatcher.findServiceActivators(MessageDispatcher.java:99) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.management.bus.MessageDispatcher.afterPropertiesSet(MessageDispatcher.java:69) ~[ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335) ~[spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:955) [spring-beans-5.3.30.jar:5.3.30]
      at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:921) [spring-context-5.3.30.jar:5.3.30]
      at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583) [spring-context-5.3.30.jar:5.3.30]
      at nl.nn.adapterframework.lifecycle.IbisApplicationContext.createApplicationContext(IbisApplicationContext.java:109) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:116) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:96) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at nl.nn.adapterframework.lifecycle.IbisApplicationServlet.init(IbisApplicationServlet.java:80) [ibis-adapterframework-core-7.9-20231028.143509.jar:7.9-20231028.143509]
      at javax.servlet.GenericServlet.init(GenericServlet.java:158) [servlet-api.jar:4.0.FR]
      at org.apache.catalina.core.StandardWrapper.initServlet(StandardWrapper.java:1106) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardWrapper.loadServlet(StandardWrapper.java:1063) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardWrapper.load(StandardWrapper.java:960) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardContext.loadOnStartup(StandardContext.java:4673) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:4980) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:683) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:658) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:662) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1184) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1932) [catalina.jar:9.0.74]
      at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_392]
      at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_392]
      at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.74]
      at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:112) [?:1.8.0_392]
      at org.apache.catalina.startup.HostConfig.deployDirectories(HostConfig.java:1094) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:476) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1617) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:318) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:898) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:795) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1332) [catalina.jar:9.0.74]
      at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1322) [catalina.jar:9.0.74]
      at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_392]
      at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75) [tomcat-util.jar:9.0.74]
      at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:134) [?:1.8.0_392]
      at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:871) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:249) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardService.startInternal(StandardService.java:428) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:917) [catalina.jar:9.0.74]
      at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [catalina.jar:9.0.74]
      at org.apache.catalina.startup.Catalina.start(Catalina.java:772) [catalina.jar:9.0.74]
      ... suppressed 4 lines
      at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:347) [bootstrap.jar:9.0.74]
      at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:478) [bootstrap.jar:9.0.74]
2023-11-13 11:50:03,978 INFO [main] {} APPLICATION - Created IbisApplicationContext [Frank2SittardGeleen] in 1661 ms
2023-11-13 11:50:04,828 WARN [main] {} digester.DigesterRulesParser - pattern [*/sender] already parsed
2023-11-13 11:50:08,769 INFO [main] {} APPLICATION - Configuration [IAF_Util] [null] startup in 4304 ms
2023-11-13 11:50:08,851 WARN [main] {} digester.DigesterRulesParser - pattern [*/sender] already parsed
2023-11-13 11:50:12,117 INFO [main] {} APPLICATION - Configuration [Common of SittardGeleen] [null] startup in 3276 ms
2023-11-13 11:50:12,179 WARN [main] {} digester.DigesterRulesParser - pattern [*/sender] already parsed
2023-11-13 11:50:12,368 INFO [main] {} APPLICATION - Configuration [Ftp] [null] startup in 199 ms
2023-11-13 11:50:12,370 INFO [main] {} APPLICATION - Application [Frank2SittardGeleen] startup in 10054 ms
2023-11-13 11:50:12,371 INFO [main] {} APPLICATION - Initialized IbisApplicationServlet`

**Comment by nielsm5** [comment]
En als je de gehele naam opgeeft? (wat volgens mij wel vereist is)

**Comment by checkiecheck** [comment]
dat had ik ook al geprobeerd en leek geen effect te hebben. Voor de zekerheid nog eens gedaan en nu is de exception weg. Zal we typefout hebben gemaakt eerder
flow.generator=nl.nn.adapterframework.util.flow.MermaidFlowGenerator
dan wordt idd class geaccepteerd.
Nu nog de plaatjes, generate button levert nog geen image.

**Comment by Matthbo** [comment]
I can confirm I might have forgotten to import the controller for the modal, I converted it to a component but the angular.js plugin to load modals (like the mermaid flow modal) can only work with controllers which is why I have both versions but forgot to import the controller after deleting the import for the component.

Tldr; will be fixed in the upgraded Console

**Comment by Matthbo** [comment]
Looks like the endpoint to retrieve the mermaid flow data is not giving back the right response in the latest nightly (v8.0).

In 7.8 and in 7.9 using the upgraded frontend it will give back http code 200 with a body containing graph data, which is incorrect/has a syntax error according to mermaid (also tested using the graph data in the [mermaid live editor](mermaid.live)).

In the latest nightly build this seems even more broken as the entire flow endpoint doesnt even return http code 200 and graph data anymore, instead it gives back 204 No Content, so I can't even test @lenardvandermaas's latest change to the MermaidFlowGenerator

**Comment by Matthbo** [comment]
Closing this because it has been solved a while ago and improved upon recently
